Assets/MixedRealityToolkit/Utilities/MixedRealityPlayspace.cs

```diff
@@ -200,7 +200,10 @@ private static void EditorSceneManagerSceneOpened(Scene scene, OpenSceneMode mod
             }
             else
             {
-                SearchForAndDisableExtraPlayspaces(scene.GetRootGameObjects());
+                if (scene.isLoaded)
```

Comment:
Do you think this should even be a little higher, alongside the `if (Application.isPlaying)` check? I wonder if we should be doing anything at all in this method if the scene isn't actually loaded. @Railboy?

https://github.com/microsoft/MixedRealityToolkit-Unity/blob/81383453589488e2df92d54c3938cab5e57bc424/Assets/MixedRealityToolkit/Utilities/MixedRealityPlayspace.cs#L192

Reply:
Good point. I was just going for a minimum-impact change.

Reply:
I believe @keveleigh is correct here. Let's move the check to line 192.

Nice fix!

Reply:
Actually, EditorSceneUtils.GetRootGameObjectsInLoadedScenes() iterates over all open scenes but skips those which are not yet loaded. So I'm thinking that maybe the check should stay where it is now (?) In other words - the fact that the scene the event pertains to isn't loaded, won't affect the path where mixedRealityPlayspace is null.